### PR TITLE
rename scientific_names.scientific_name to scientific_names.name

### PR DESCRIPTION
**Pull request by maco**
this mirrors alternate_names.name and allows making code more DRY

### CODE REVIEW

File: app/models/crop.rb [hunk 1]
```diff
@@ -139,11 +137,11 @@ def default_scientific_name
   # later we can choose a default photo based on different criteria,
   # eg. popularity
   def default_photo
-    return photos.first if photos.any?
```
Comment (CloCkWeRX) [inline app/models/crop.rb]: We want this return here, otherwise it'll keep going to the next line

File: db/schema.rb [hunk 2]
```diff
@@ -433,15 +433,15 @@
...
-  create_table "scientific_names", force: true do |t|
-    t.string   "scientific_name", null: false
-    t.integer  "crop_id",         null: false
+  create_table "scientific_names", force: :cascade do |t|
+    t.string   "name",       null: false
+    t.integer  "crop_id",    null: false
     t.datetime "created_at"
     t.datetime "updated_at"
     t.integer  "creator_id"
   end
```
Comment (CloCkWeRX) [inline db/schema.rb]: Its interesting the lack of indexes on crop_id, etc in the schema. Should fix that in another ticket probably

File: app/controllers/crops_controller.rb [hunk 3]
```diff
@@ -169,19 +169,15 @@ def update
...
           params[:alt_name].each do |index, value|
-            alt_name = @crop.alternate_names.create(name: value, creator_id: current_member.id)
-          end
-
-          @crop.scientific_names.each do |sci_name|
-            sci_name.destroy
+            create_name('alternate', value)
           end
+        end
+        if !params[:sci_name].nil?
```
Comment (Br3nda) [inline app/controllers/crops_controller.rb]: Could `unless`be better than `if!`

**Comment by pozorvlak** [comment]
@Br3nda: you're right, the "what the reviewers think" summary view is really useful!

**Comment by maco** [comment]
Since @Br3nda says 👍 and I fixed the things @CloCkWeRX mentioned, I'm going to merge it.

**Pull request merged**